vademecum.md

```diff
@@ -83,7 +83,7 @@ Grazie a questa estensione è possibile pulire la cronologia di navigazione peri
 
 ### Altri progetti Mozilla
 
-Numerosi sono i progetti portati avanti dalla Mozilla Foundation e tra quelli non elencati ci sono altri innumerevoli progetti, come MDN, Open Leadership e Internet Health Report. Tutti, comunque, si rivolgono a persone con differenti competenze e hanno tutti un fattore in comune: la visione del **Web come risorsa aperta a chiunque**. In dettaglio i progetti più considerevoli di Mozilla:
+Oltre a Firefox La Mozilla foundation porta avanti numerosi progetti in ambiti diversi ma tutti con l'obiettivo di avere il  **Web come una risorsa aperta a chiunque**. Tra questi ci sono iniziative come MDN(*M*ozilla *D*eveloper *N*etwork), Open Leadership, Internet Health Report, Common Voice. In seguito altri progetti in dettaglio:
```

Comment:
Personalmente preferisco la "mia" descrizione...

vorrei altri pareri... @Mte90 cosa ne pensi tu?

Reply:
la parte iniziale "Oltre a Firefox" non mi suona bene come anche "In seguito altri progetti" rileggendolo.
Per il resto mi sembra una semplificazione che non ci sta male.

Reply:
Come volete :)
Possiamo anche provare a "unire" le versioni, quindi a rielaborarle entrambe :)

Reply:
Ditemi voi